qhub_ops/cli/__init__.py

```diff
@@ -1,16 +1,26 @@
 import argparse
-import sys
 import logging
+import sys
+from os import path
 
 from qhub_ops.cli.deploy import create_deploy_subcommand
 from qhub_ops.cli.render import create_render_subcommand
 
+here = path.abspath(path.join(path.dirname(__file__), "../.."))
```

Comment:
`here` is a bit misleading, it is referring to project root directory

Reply:
yeah. let me rename it to root.